Assets/HoloToolkit/Utilities/Editor/BuildCommands.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Implements functionality for building HoloLens applications
+    /// </summary>
+    public static class BuildCommands
+    {
+        public static readonly string BuildLocation = "Builds/HoloLens";
+
+        /// <summary>
+        /// Do a build configured for the HoloLens, returns the error from BuildPipeline.BuildPlayer
+        /// </summary>
+        public static string BuildForHololens()
+        {
+            var scenes = EditorBuildSettings.scenes.Where(s => s.enabled).Select(s => s.path);
+
+            // Cache the current settings
+            BuildTarget oldBuildTarget = EditorUserBuildSettings.activeBuildTarget;
+            WSASDK oldSDK = EditorUserBuildSettings.wsaSDK;
+            WSAUWPBuildType oldBuildType = EditorUserBuildSettings.wsaUWPBuildType;
+
+            // Set the sdk and build type
```

Comment:
The BuildPipeline.BuildPlayer unity command is doing the platform switch, and leaving it in the BuildTarget after complete so I'm just setting it back. If you think it'd be more clear to be explicit I can do that, though it might be confusing when someone removes the explicit line and things still change.